tests/e2e/verfication_post_day1_test.go

```diff
@@ -176,8 +176,9 @@ var _ = Describe("TF Test", func() {
 					err = importService.Import(importParam)
 					Expect(err.Error()).To(ContainSubstring("Cannot import non-existent remote object"))
 
-					// skip due to bug :: OCM-5246
-					// Expect(output).To(ContainSubstring(profile.ComputeMachineType))
+					By("remove .tfstate file to prevent destructive outcomes")
```

Comment:
I don't  think we should use words like destructive - regular clean up(revert) after the test execution)

Reply:
fixed :)